include/nbl/asset/utils/CCPUMeshPackerV1.h

```diff
@@ -156,13 +111,18 @@ class CCPUMeshPackerV1 final : public IMeshPacker<ICPUMeshBuffer, MDIStructType>
 		MeshPackerConfigParams<Iterator>* packerParamsOut);
 
 	//! shrinks byte size of all output buffers, so they are large enough to fit currently allocated contents. Call this function before `instantiateDataStorage`
-	void shrinkOutputBuffersSize() // TODO: shrink isn't special, can only happen before data storage is instantiated, so do it in the base class
+	virtual void shrinkOutputBuffersSize() override
 	{
-		m_allocParams.MDIDataBuffSupportedCnt = m_MDIDataAlctr.safe_shrink_size(0u, 1u);
-		m_allocParams.perInstanceVertexBuffSupportedByteSize = m_perInsVtxBuffAlctr.safe_shrink_size(0u, 1u);
-		m_allocParams.indexBuffSupportedCnt = m_idxBuffAlctr.safe_shrink_size(0u, 1u);
-		m_allocParams.vertexBuffSupportedByteSize = m_vtxBuffAlctr.safe_shrink_size(0u, 1u);
-		// TODO: SHRINK ACTUAL ALLOCATORS AND THEIR RESERVED SPACES! (CREATE NEW ALLOCATORS WITH NEW RESERVED SPACES, THEN DELETE OLD RESERVED, THEN SWAP ALLOCATOR AND RESERVED MEMBERS) 
+		base_t::shrinkOutputBuffersSize();
+
+		uint32_t mdiDataBuffNewSize = m_perInsVtxBuffAlctr.safe_shrink_size(0u, alctrTraits::max_alignment(m_perInsVtxBuffAlctr));
```

Comment:
`mdiDataBuffNewSize` is a really bad variable name here!

shouldn't it be `perInstanceDataBuffNewSize` ?

Reply:
oops